internal/service/repository_test.go

```diff
@@ -16,14 +16,18 @@ type RepositoryStore struct {
 	store.Store
 	RepositoryVal api.Repository
 	EventVal      api.Event
+	events        []api.Event
```

Comment:
Should events be emitted based on the sync status of a repository?

I couldn't see one after I created a repository with an invalid URL and the sync status becomes invalid.